tests/llmcompressor/modeling/test_calib_qwen3_5_moe.py

```
@@ -0,0 +1,171 @@
+        moe_layer = None
+        for _, module in model.named_modules():
+            if isinstance(module, CalibrationQwen3_5MoeSparseMoeBlock):
+                moe_layer = module
+                break
+
+        assert moe_layer is not None
+
+        num_experts = len(moe_layer.experts)
+        expert_triggered = [False for _ in range(num_experts)]
+
+        def hook_fn(i, module, input, output):
```

Ruff (0.15.9)

[error] Function argument `input` is shadowing a Python builtin

(A002)